app/models/canonical/sync/association_syncer.rb

```diff
@@ -4,7 +4,7 @@ module Canonical
   module Sync
     class AssociationSyncer < Syncer
       def perform
-        raise Canonical::Sync::PrerequisiteNotMetError.new(prerequisite_error_message) unless prerequisite_conditions_met?
+        raise PrerequisiteNotMetError.new(prerequisite_error_message) unless prerequisite_conditions_met?
```

Comment:
We don't really need the fully qualified name here, especially since the line is already on the long side.